bot.py

```diff
@@ -251,7 +313,117 @@ def get_members_by_role(role):
+                        else:
+                            #Zero-length messages cannot be quotes
+                            msg = "Zero-length messages cannot be quotes. Duh."
+                            log.info(msg)
+                            await client.send_message(message.channel, msg)
+                            return
+                #No messages found in channel by specified author
+                msg = "No recent messages by {} exist in {}."
+                log.info(msg.format(message.mentions[0].name, message.channel))
+                await client.send_message(message.channel, msg.format(message.mentions[0].name, message.channel.mention))
+                return
+        else:
+            #Too many or no mention provided
+            msg = "You must provide only 1 user mention. Not {}, dumbass.".format(len(message.mentions))
+            log.info(msg)
+            await client.send_message(message.channel, msg)
+            return
+    
+    #Draw and post image from inspirational quote database
+    if message.content.lower().startswith("!inspire"):
+        await client.send_typing(message.channel)
+        quotes = get_quotes()
+        if len(quotes) < 1:
+            #No quotes found
+            msg = "No inspirational quotes found. Not surprising honestly."
+            log.error("No quotes found")
+            await client.send_message(message.channel, msg)
+            return
+        quote = random.choice(quotes)
+        content = quote.content
+        if len(quote.mentions) > 0:
+            for member in quote.mentions:
+                content = content.replace(member.mention, member.name)
+        quote_image = create_quote_image(content, quote.author.name)
+        await client.send_file(message.channel, os.path.join(sys.path[0], quote_image), filename=None, tts=False)
+        os.remove(os.path.join(sys.path[0], quote_image))
+        return
 
+    #View quotes (super_waifus)
+    if message.content.lower().startswith("!viewquotes"):
+        if not is_super_waifu(member):
+            msg = "{user}, you are not authorized to do that!"
+            await client.send_message(message.channel, msg.format(user=member.mention))
+            return
+        quotes = get_quotes()
+        if len(quotes) < 1:
+            msg = "No quotes found."
+            log.error(msg)
+            await client.send_message(message.channel, msg)
+            return
+        reply_msg = ""
+        for i, quote in enumerate(quotes):
+            quote_info = "ID: {}\nAuthor: {}\nQuote: {}".format(quote.id, quote.author.name, quote.content)
+            if len(quote_info) > 190:
+                reply_msg += quote_info[:187] + "...\n\n"
+            else:
+                reply_msg += quote_info + "\n\n"
+            if (i%10 == 0 and i != 0) or i == len(quotes) - 1:
+                await client.send_message(message.channel, reply_msg)
+                await asyncio.sleep(3)
+                reply_msg = ""
+                
```

Comment:
There should be a return here, outside of the for loop.

Reply:
Yup.